### ISSUE TITLE
Read-Only property : does it work?

### ISSUE THREAD

**Issue by Vincebml** [Mermade/widdershins#84]
Hi,

I've written the following OpenAPI 3 partial YAML:

```yaml
...
components:
  schemas:
    User:
      type: object
      properties:
        resource:
          type: string
          readOnly: true
          example: user
          default: user
          description: String representing the resource’s type.
        created_at:
          type: string
          readOnly: true
          format: date-time
          description: Time at which the resource was created.
        updated_at:
          type: string
          readOnly: true
          format: date-time
          description: Time at which the resource was updated.
        username:
          type: string
          minLength: 3
          maxLength: 64
          example: jdoe
          description: The user’s username.
        email:
          type: string
          format: email
          example: jdoe@gmail.com
          description: The user’s email address.
      required:
        - username
        - email
...
```

Then, I've executed the following command:

```shell
widdershins swagger/api.yaml index.html.md
```

Here is the generated output (for the endpoint to update the authenticated user):

`PUT /user/`

Update the current authenticated user (you).

> Body parameter

```json
{
  "resource": "user",
  "created_at": "2018-02-13T15:18:13Z",
  "updated_at": "2018-02-13T15:18:13Z",
  "username": "jdoe",
  "email": "jdoe@gmail.com"
}
```

I've added the `readOnly: true` property so 'resource', 'created_at' and 'updated_at' fields should not be displayed. But they are displayed... Do you know why this is happening? How to fix it?

Thank you!

**Comment by MikeRalphson** [comment]
Hi, thanks for pointing this out. I've added the missing `skipReadOnly` and `skipWriteOnly` options to `openapi-sampler` where applicable which should fix the examples.

This fix is in v3.0.3

**Comment by Vincebml** [comment]
Thank you, Mike!
